packages/cozy-procedures/src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Route, IndexRoute } from 'react-router'
+
+import Procedure from './Procedure'
+import Overview from './components/OverviewContainer'
+import PersonalDataForm from './components/PersonalDataFormContainer'
+import OtherForm from './components/OtherComp'
+
+const injectProcedureRoutes = ({ root }) => (
+  <Route path={root} component={Procedure}>
+    <IndexRoute component={() => <Overview />} />
```

Comment:
I can't figure that one out — any component wrapped in a `connect()` HOC can't be passed here with a simple `component ={Overview}`. But wrapped in a function and it works..?

Reply:
Just spent some time trying to figure it out. It seems that redux's connect "hides" the fact that `Overview` is a react component and has a `render()` and so react-router is not happy. I haven't found a way to fix it :-(